Assets/Plugins/Source/Core/Utility/FileUtility.cs

```diff
@@ -22,8 +22,12 @@
 
 namespace PlayEveryWare.EpicOnlineServices.Utility
 {
+    using Extensions;
     using System;
+    using System.Collections.Generic;
     using System.IO;
+    using System.Linq;
```

Comment:
Danger, System.Linq spotted in runtime code! Is this an acceptable use of it?

Reply:
It is a reasonable question to ask, and to be more clear it should be guarded against, but the places where the `System.Linq` namespace is actually utilized are within functions that should only ever be called when in runtime. I'll adjust things to guarantee that is the case and ping you back once it's fixed.